test/controllers/BankDetailsControllerSpec.scala

```diff
@@ -220,12 +237,64 @@ class BankDetailsControllerSpec extends ItSpec {
         ("isSoleSignatory", "Yes")
       )
       val expectedContentAndHref: List[(String, String)] = List(
-        ("Sort code must be 6 digits", sortCodeFieldId),
-        ("Account number must be between 6 and 8 digits", accountNumberFieldId)
+        ("Sort code must be 6 digits", EnterDirectDebitDetailsPage.sortCodeFieldId),
+        ("Account number must be between 6 and 8 digits", EnterDirectDebitDetailsPage.accountNumberFieldId)
       )
       testFormError(formData: _*)(expectedContentAndHref)
       EssttpBackend.DirectDebitDetails.verifyNoneUpdateDirectDebitDetailsRequest(TdAll.journeyId)
     }
   }
 
+  "GET /check-your-direct-debit-details should" - {
+    "return 200 and the check your direct debit details page" in {
+      AuthStub.authorise()
+      EssttpBackend.DirectDebitDetails.findJourneyAfterUpdateDirectDebitDetails()
+
+      val fakeRequest = FakeRequest().withAuthToken().withSession(SessionKeys.sessionId -> "IamATestSessionId")
+      val result: Future[Result] = controller.checkBankDetails(fakeRequest)
+
+      status(result) shouldBe Status.OK
+      contentType(result) shouldBe Some("text/html")
+      charset(result) shouldBe Some("utf-8")
+
+      val pageContent: String = contentAsString(result)
+      val doc: Document = Jsoup.parse(pageContent)
+
+      doc.title() shouldBe ConfirmDirectDebitDetailsPage.expectedPageTitle
+      doc.select(".govuk-heading-xl").text() shouldBe ConfirmDirectDebitDetailsPage.expectedH1
+      doc.select(".hmrc-header__service-name").text() shouldBe expectedServiceName
+      doc.select(".hmrc-sign-out-nav__link").attr("href") shouldBe "http://localhost:9949/auth-login-stub/session/logout"
+      doc.select("#back").attr("href") shouldBe routes.BankDetailsController.enterBankDetails().url
+
+      val summaries = doc.select(".govuk-summary-list").select(".govuk-summary-list__row").iterator().asScala.toList
+      summaries.size shouldBe 3
+
+      val expectedAccountNameRow = SummaryRow("Account name", "Bob Ross", routes.BankDetailsController.enterBankDetails().url)
+      val expectedSortCodeRow = SummaryRow("Sort code", "123456", routes.BankDetailsController.enterBankDetails().url)
+      val expectedAccountNumberRow = SummaryRow("Account number", "12345678", routes.BankDetailsController.enterBankDetails().url)
+      val expectedSummaryRows = List(expectedAccountNameRow, expectedSortCodeRow, expectedAccountNumberRow)
+      extractSummaryRows(summaries) shouldBe expectedSummaryRows
+
+      //assert direct debit content
```

Comment:
is this something you forgot to do or intended to do in the future?

Reply:
Yeah, I forgot to do this but already updated in most recent commit